package.json

```diff
@@ -42,7 +42,6 @@
     "leche": "^2.3.0",
     "lodash": "^4.17.19",
     "mocha": "^7.1.0",
-    "openzeppelin-test-helpers": "^0.5.1",
```

Comment:
why it is removed?

Reply:
no longer needed since I'm testing for the event elsewhere